python/pyspark/sql/functions.py

```diff
@@ -2519,6 +2519,9 @@ def to_utc_timestamp(timestamp: "ColumnOrName", tz: "ColumnOrName") -> Column:
 
 def timestamp_seconds(col: "ColumnOrName") -> Column:
```

Comment:
Let's fix functions.scala and functions.R together since we're here.

Reply:
I aligned the docstring in R (+ added an example) and Scala. Since the Scala functions do not seem to feature any examples, I did not add one there.